src/fromager/__main__.py

```diff
@@ -79,6 +79,12 @@
     help="control removal of working files when a build completes successfully",
 )
 @click.option("--variant", default="cpu", help="the build variant name")
+@click.option(
+    "-n",
+    "--number-of-jobs-available",
```

Comment:
I recommend a shorter name. `-j`, `--jobs`, and `--parallel`  are common options used by `make` and `cmake`. A long name also makes it harder to set the option with an env var. `FROMANGER_NUMBER_OF_JOBS_AVAILABLE=8` is a mouthful. :)

```suggestion
    "-j",
    "--jobs",
```